src/ray/raylet/format/node_manager.fbs

```diff
@@ -69,8 +69,6 @@ enum MessageType:int {
   // before the checkpoint, and make any tasks on the frontier runnable by
   // making their execution dependencies available.
   SetActorFrontier,
```

Comment:
Can we also remove the ActorFrontier stuff above?

Reply:
Removed `UpdateActorFrontier` (method that doesn't have cc body), `SetActorFrontier`, `GetActorFrontierReply`, `GetActorFrontierRequest` (flat buffer)